react/components/Pagination/PageIndicator.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function PageIndicator({ currentItemFrom, itemTo, textOf, totalItems }) {
+  return (
+    <div className="c-muted-2 t-small">
+      {`${currentItemFrom} - ${itemTo} `}
```

Comment:
```suggestion
      {`${currentItemFrom} - ${itemTo}`}
```
Does the space here is something that you want?

Reply:
Yes, it adds a space between `itemTo` and `textOf`